.github/workflows/test.yml

```diff
@@ -5,6 +5,7 @@ on:
 
 jobs:
   test:
-    uses: UKHomeOffice/hocs-github-actions/.github/workflows/test-gradle.yml@v2
+    uses: UKHomeOffice/sas-github-workflows/.github/workflows/test-gradle.yml@v1
     with:
-      components: 'localstack'
+      dockerComposeComponents: 'localstack'
```

Comment:
Feels like this could have a better name like `dockerServices` or something but it's clear...